src/uptane/tufrepository.cc

```diff
@@ -183,15 +183,6 @@ void TufRepository::initRoot(const Json::Value& new_content) {
                                                                << requiredThreshold);
       throw IllegalThreshold(name_, "root.json contains a role that requires too many signatures");
     }
-    for (Json::ValueIterator it_keyid = (*it)["keyids"].begin(); it_keyid != (*it)["keyids"].end(); ++it_keyid) {
```

Comment:
`Every public key has a unique identifier (PublicKey.keyid). This identifier MAY be, for example, the SHA-256 hash of the public key." - from the UPTANE impementation spec.`

So?